.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Thank you for your interest in contributing to the venia-pwa-concept project! Before you start contributing, please take a moment to read through the following guidelines:
+
+* [Code of Conduct]
+* [Support]
+
+To contribute to this repository, start by forking the [official repository] and following the installation instructions in the README file.
+
+## Pull Request checklist
+
+* Code contributions must adhere to the [Coding style guide](#coding-style-guide)
+* PR commits should contain [meaningful commit messages]
+* To help with reviews, your PR should only create/revise a single feature or fix a single issue.
+* If your PR fixes a bug, please provide a step-by-step description of how to reproduce the bug.
+* If your PR addresses an existing issue, please reference that issue in the title or description.
+
+### Coding style guide
+
+**TODO: Write style guide**
+
+## Contribution process
+
+**TODO: Write contribution process**
+
+## Report an issue
+
+Create a [GitHub issue] and put an **X** in the appropriate box to report an issue with the project.
+Provide as much detail as you can in each section to help us triage and process the issue.
+
+### Issue types
+
+* Bug - An error, flaw, or failure in the code
+* Feature suggestion - A missing feature you would like to see implemented in the project
+* Other - Any other type of task related to the project
+
+**Note:**
+***DO NOT** create GitHub issues asking for help on bugs in your project or other support-type questions.*
```

Comment:
Can we make this language a bit more gentle? A bold "DO NOT" sounds like we're frustrated after an onslaught of irrelevant issues. How about:

> Please avoid creating GitHub issues asking for help on bugs in your project that are outside the scope of this project.